content/docs/1.3.0/advanced-resources/deploy/customizing-default-settings.md

```diff
@@ -181,6 +189,17 @@ But please be aware **this will bypass Longhorn backend validation**.
 kubectl -n longhorn-system get configmap longhorn-default-setting -ojsonpath="{.data['default-setting\.yaml']}" | awk '{if ($2) print $0;}' | while read key val; do kubectl -n longhorn-system patch settings ${key%:} --type='json' -p='[{"op": "replace", "path": "/value", "value": "'${val}'"}]'; done
```

Comment:
Do we still need `using script`? This should be fixed in 1.3.0.

Reply:
Yes. This the old way, forgot to remove it.